src/C4Config.cpp

```diff
@@ -663,10 +722,9 @@ void C4ConfigControls::ResetKeys()
 	StdCompilerNull Comp; Comp.Compile(mkParAdapt(*this, true));
 }
 
-const char *C4Config::AtExeRelativePath(const char *szFilename)
-{
-	// Specified file is located in ExePath: return relative path
-	return GetRelativePathS(szFilename, General.ExePath);
+const char *C4Config::AtExeRelativePath(const char *szFilename) {
```

Comment:
The style guide mandates a specific brace style, and this change doesn't even do anything else.